sphinx_sitemap/__init__.py

```diff
@@ -142,7 +148,7 @@ def create_sitemap(app, exception):
                     lang=lang, version=version, link=link
                 ))
 
-    filename = app.outdir + "/sitemap.xml"
+    filename = app.outdir + "/" + app.config.sitemap_name
```

Comment:
This would need the new config value `sitemap_filename`.
```suggestion
    filename = app.outdir + "/" + app.config.sitemap_filename
```